grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -510,6 +513,177 @@ class ApiController extends BaseController {
         }
     }
 
+    @GrantAccess
+    @Path("/api/opus/{opusId}/profile/{profileId}/image/{imageId}")
+    @Operation(
+            summary = "Get images associated with a profile",
+            operationId = "/api/opus/{opusId}/profile/{profileId}/image/{imageId}",
+            method = "GET",
+            responses = [
+                    @ApiResponse(
+                            responseCode = "200",
+                            content = @Content(
+                                    mediaType = "application/json",
+                                    schema = @Schema(
```

Comment:
should be - schema: 
               type: string
               format: binary

Reply:
Changed